qutebrowser/app.py

```diff
@@ -723,6 +726,7 @@ def _shutdown(self, status, restart):
         # Now we can hopefully quit without segfaults
         log.destroy.debug("Deferring QApplication::exit...")
         objreg.get('signal-handler').deactivate()
+        objreg.get('session-manager').session_delete('_autosave', force = True)
```

Comment:
Please don't uses spaces around `=` for function arguments (see [this section](https://www.python.org/dev/peps/pep-0008/#other-recommendations) of pep8)